config/initializers/raven.rb

```diff
@@ -8,6 +8,7 @@
   config.processors << Sentry::Processor::EmailSanitizer
   config.processors << Sentry::Processor::PIISanitizer
   config.processors << Sentry::Processor::LogAsWarning
+  config.processors << Sentry::Processor::FilterRequestBody
```

Comment:
not sure i understand your question, `['request']['data']` is where sentry stores the request body. every sentry log not from sidekiq includes a request key, and every request that has a request body includes the `['request']['data']` key

Reply:
to clarify, `['request']` is the not the request body and it has a key named `data`, `['request']['data']` is the full request body

Reply:
So this is sanitizing the data in the request, right? Is there a clearer name to communicate what it does? PIIBodySanitizer? 

Not directly an issue here, but seems like PIISanitizer should handle this too. But I get you're focusing on a single controller in this instance. I do think we'd likely want it in all controllers but I'm not positive.

Reply:
this is filtering all request bodies for specific controllers regardless of whether or not there is PII in them, unlike PIISanitizer which filters only if certain PII keys are present. making the name similar to PIISanitizer sounds like it might cause confusion about what this class is doing.